code/__DEFINES/anomaly.dm

```diff
@@ -0,0 +1,40 @@
+/**
+ *  # Anomaly Defines
+ *  This file contains defines for the random event anomaly subtypes.
+ */
+
+///Time in ticks before the anomaly goes poof/explodes depending on type.
+#define ANOMALY_COUNTDOWN_TIMER (75 SECONDS)
+
+/**
+ * Nuisance/funny anomalies
+ */
+
+///Time in ticks before anomaly spawns
```

Comment:
This presumably means subsystem ticks and not world ticks, which are totally different. This should all be seconds. :(

Reply:
Okay, will do